src/services/leafletLayerHelpers.js

```diff
@@ -55,6 +55,14 @@ angular.module('ui-leaflet')
                 return L.tileLayer(url, params.options);
             }
         },
```

Comment:
AHHH! Thanks @elesdoar , he is correct this can't be done with the current design for this factory. So I will merge. Anyway what we were talking about is the goal. At least we know what we want to work towards .